src/core/config/tabular_data/input_table_collection_type.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <memory>
+
+#include "input_table_type.h"
+#include "model/table/idataset_stream.h"
```

Comment:
Don't forget to fix includes:
```suggestion
#include <vector>

#include "input_table_type.h"
```